internal/cmd/fga.go

```diff
@@ -65,6 +66,15 @@ func init() {
 	queryCmd.Flags().String("order", "", "order in which a list of results should be returned (asc or desc)")
 	fgaCmd.AddCommand(queryCmd)
 
+	// schema
+	convertSchemaCMD.Flags().String("to", "json", "output to (schema or json)")
+	convertSchemaCMD.Flags().String("output-format", "pretty", "output format (pretty or raw). use raw for machine-readable output or writing to a file")
```

Comment:
Thoughts on making this a boolean flag called `--raw`?

Reply:
That's what I originally named it 😃 Yes, I can change it back. I originally thought it was a bit ambiguous.

Reply:
Alternatively, we could shorten it to just `--output`:
```shell
--output raw
--output pretty
```